doc/design.txt

```diff
@@ -24,7 +24,7 @@
 The Mynewt bootloader comprises two packages:
```

Comment:
If you're updating this to note that there are multiple ports, how about saying "mcuboot" instead of "The Mynewt bootloader"?